diff --git a/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java b/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java
index c362e6a..87e890e 100644
--- a/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java
+++ b/node_modules/onnxruntime-react-native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java
@@ -48,10 +48,11 @@ public class OnnxruntimeModule extends ReactContextBaseJavaModule {
       System.loadLibrary("onnxruntimejsi");
       JavaScriptContextHolder jsContext = getReactApplicationContext().getJavaScriptContextHolder();
       CallInvokerHolderImpl jsCallInvokerHolder =
-        (CallInvokerHolderImpl) getReactApplicationContext().getCatalystInstance().getJSCallInvokerHolder();
+        (CallInvokerHolderImpl) getReactApplicationContext().getJSCallInvokerHolder();
       nativeInstall(jsContext.get(), jsCallInvokerHolder);
       return true;
     } catch (Exception e) {
+      android.util.Log.e("Onnxruntime", "Failed to install ONNX Runtime JSI bindings", e);
       return false;
     }
   }
